src/veh_interact.h

```diff
@@ -77,6 +77,7 @@ class veh_interact
         int name_w;
 
         vehicle *veh;
+        bool has_screwdriver;
```

Comment:
I missed the detail about the small motor, that one should be fine, just
not alternators.